### DISCUSSION TITLE
Approve and run shouldn't show if I can't approve

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#52227] [Mobile] [answered]
GitHub Mobile beta

https://github.com/argoproj/argo-ui/pull/352

![Screenshot_20230407-010626.png](https://user-images.githubusercontent.com/2119212/230545233-6b5cc0ec-0f2d-4c5f-8cec-99db61eacb70.png)

I clicked the button because it was there. It almost certainly isn't there in a browser view and it shouldn't be here either.

**Comment by stevepopovich** [comment, marked as answer]
I logged this with the team. Thanks.
